src/strategies/deno.ts

```diff
@@ -81,8 +81,6 @@ export default class DenoTsStrategy extends VersionStrategy {
         ? tag.slice(args.versionPrefix.length).trim()
         : tag.trim();
     }
-    throw new Deno.errors.NotFound(
-      'Cannot find version export in ' + file + 'or fallback git tags',
-    );
+    return '';
```

Comment:
I'm unsure how to feel about this change. For some reason the default version in the Deno one isn't supplied, but it ets populated in the actual CLI run. Might revisit this in perhaps a follow up CR